litmus-portal/frontend/src/components/ReliabilityScore/index.tsx

```diff
@@ -3,7 +3,7 @@ import React, { useState } from 'react';
 import CustomSlider from '../CustomSlider';
 import CustomResultModal from '../ResultModal';
 import useStyles from './styles';
-import ButtonOutlineIcon from '../ButtonOutlineIcon';
+import ButtonOutline from '../Button/ButtonOutline';
 import InfoTooltip from '../InfoTooltip';
 
 const ReliablityScore = () => {
```

Comment:
Why is the state variable taking number or array of numbers? It should just be a no. representing the reliability weight?

Reply:
The Pretto Slider takes in a number | array of numbers and maps it in 'marks' which is rendered. In this case from 1 to 10. The reliability score reuses that component so there is an onChangeCommited property which accepts an array of numbers.